### DISCUSSION TITLE
how to setup local database in electron vite like sqlite3

### DISCUSSION THREAD

**Discussion by travis2319** [alex8088/electron-vite#677] [General]
i didnt find anything online so asking you'll how to do it 
i am using react template with ts enabled

**Comment by hyusetiawan** [comment]
sqlite can only be accessed through the main process, in the main process it's essentially a node environment so you can follow any guide on sqlite for that

**Reply by travis2319** [reply]
thanks , actually sqlite3 does connect with electron easily because its a npm package so for that just need to convert sqlite to .node file then it works for me using npx electron-rebuild
